docs/changelog.md

```diff
@@ -1,3 +1,7 @@
+0.15.2.0
```

Comment:
Actually the entry for 0.15.1.4 is unreleased, these changes and other changes in the changelog for 0.15.1.4 should be combined into one section

Reply:
Done in b677c5d and conflict merged in fa5397b